api/src/utils/auth.py

```
@@ -25,104 +25,108 @@ class AuthenticationError(Exception):
         response = secrets_client.get_secret_value(SecretId=secret_name)
         psk = response['SecretString']
-        
+
         # Cache the value
         _psk_cache[cache_key] = (psk, current_time)
-        
+
         return psk
-        
+
     except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)